Win32.xs

```diff
@@ -1667,6 +1667,10 @@ XS(w32_GetProcessPrivileges)
     XSRETURN(1);
 }
 
+#ifndef RRF_SUBKEY_WOW6464KEY
+#  define RRF_SUBKEY_WOW6464KEY 0x00010000
+#endif
```

Comment:
Could you please move the preprocessor definitions to the top of the file, where all the other constants are defined? Thanks!